test/topology/test_random_tables.py

```diff
@@ -24,7 +24,8 @@ async def test_new_table(manager, random_tables):
     assert len(res) == 1
     assert list(res[0])[:2] == vals
     await random_tables.drop_table(table)
-    with pytest.raises(InvalidRequest, match='unconfigured table'):
+    # NOTE: On rare occasions the exception is ReadFailure
+    with pytest.raises((InvalidRequest, ReadFailure), match='unconfigured table'):
```

Comment:
Did these `ReadFailure` exceptions contain the `unconfigured table` string?

Reply:
Good point, they don't.

Reply:
Fixed in https://github.com/scylladb/scylladb/commit/bcc0a5cb577d66d5bbb477b969554b754b3065db